vmm/src/cpu.rs

```diff
@@ -1435,10 +1585,14 @@ impl Aml for CpuManager {
         let hid = aml::Name::new("_HID".into(), &"ACPI0010");
         let uid = aml::Name::new("_CID".into(), &aml::EISAName::new("PNP0A05"));
         // Bundle methods together under a common object
+        #[cfg(target_arch = "x86_64")]
         let methods = CPUMethods {
             max_vcpus: self.config.max_vcpus,
         };
+        #[cfg(target_arch = "x86_64")]
         let mut cpu_data_inner: Vec<&dyn aml::Aml> = vec![&hid, &uid, &methods];
```

Comment:
Ditto, I think it's fine to have the methods. We can just make them NOPs in the MMIO side or inside the methods themselves.

Reply:
Done.